src/Register.php

```diff
@@ -0,0 +1,45 @@
+<?php namespace Leean\Elements;
+
+use Leean\Elements\Options;
+
+/**
+ * Class to provide predefined ACF elements.
+ */
+class Register
+{
+	/**
+	 * The location for the elements.
+	 *
+	 * @var array
+	 */
+	private static $_location = [];
+
+	/**
+	 * Registered elements.
+	 *
+	 * @var array
+	 */
+	private static $_elements = [];
+
+	/**
+	 * Init. Takes the location and the elements to register in that location.
+	 *
+	 * @param array $location Array suitable for ACF location parameter.
+	 * @param arrat $elements Array of element names to show in this location.
+	 */
+	public static function init( $location, $elements ) {
+		self::$_location = $location;
```

Comment:
I don't get the location part - why don't we just use the same location each time? E.g. always create a page called Options/Your Organization. Contact and social could always go here.
If necessary we could add a filter to allow customization, but i don't think that will be necessary for these 2.